docs/reference/how-to/disk-usage.asciidoc

```diff
@@ -158,3 +158,27 @@ on disk usage. In particular, integers should be stored using an integer type
 stored in a `scaled_float` if appropriate or in the smallest type that fits the
 use-case: using `float` over `double`, or `half_float` over `float` will help
 save storage.
+
+[float]
+=== Use index sorting to colocate similar documents
+
+Elasticsearch compresses multiple documents at once in order to improve the
```

Comment:
Maybe something like "When Elasticsearch stores `_source`, it compresses multiple documents at once to improve the overall compression ratio...." so we don't get people thinking that doc values and the inverted index bits are stored like this.